docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -32,59 +32,67 @@ weight: 8
 
 # Manage dashboards
 
-A dashboard is a set of one or more [panels][] that visually presents your data in one or more rows.
-
-For more information about creating dashboards, refer to [Add and organize panels](../add-organize-panels).
-
 This topic includes techniques you can use to manage your Grafana dashboards, including:
 
 - [Creating and managing dashboard folders](#create-and-manage-dashboard-folders)
 - [Exporting and importing dashboards](#export-and-import-dashboards)
 - [Organizing dashboards](#organize-a-dashboard)
 - [Troubleshooting dashboards](#troubleshoot-dashboards)
 
+For more information about creating dashboards, refer to [Add and organize panels](../add-organize-panels).
+
+## Browse dashboards
+
+On the **Dashboards** page, you can browse and manage folders and dashboards. This includes the options to:
+
+- Create folders and dashboards
+- Move dashboards between folders
+- Delete multiple dashboards and folders
+- Navigate to a folder
+- Manage folder permissions. For more information, refer to [Dashboard permissions](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#dashboard-permissions).
+
+{{% admonition type="note" %}}
+As of Grafana 10.2, there is no longer a special **General** folder. Dashboards without a folder are now shown at the top level alongside folders.
+{{% /admonition %}}
+
 ## Create a dashboard folder
 
 Folders help you organize and group dashboards, which is useful when you have many dashboards or multiple teams using the same Grafana instance.
 
-**Before you begin:**
-
-- Ensure that you have Grafana Admin or Super Admin permissions. For more information about dashboard permissions, refer to [Dashboard permissions][].
+> **Before you begin:** Ensure you have Editor permissions or greater to create folders. For more information about dashboard permissions, refer to [Dashboard permissions][].
 
 **To create a dashboard folder:**
 
-1. Sign in to Grafana.
-1. Click **Dashboards** in the left-side menu.
-1. On the Dashboards page, click **New** and select **New folder** in the dropdown.
+1. Click **Dashboards** in the main menu.
+1. On the **Dashboards** page, click **New** and select **New folder** in the drop-down.
 1. Enter a unique name and click **Create**.
 
-   Do not use the name of a folder that has already been provisioned (such as "General") and avoid special characters (except underscores and hyphens).
-
 When you save a dashboard, you can either select a folder for the dashboard to be saved in or create a new folder.
 
-## Manage dashboards
+{{% admonition type="note" %}}
+Alerts cannot be placed in folders with slashes (\ /) in the name. If you wish to place Alerts in the folder, do not use slashes in the folder name.
```

Comment:
I like this edit, but should we not also have it as a note at the creation step, when you're actually setting a folder name?
```suggestion
Alerts cannot be placed in folders with slashes (\ /) in the name. If you wish to place alerts in the folder, do not use slashes in the folder name.
```

Reply:
Apologies! I was seeing the #Manage dashboards heading and not realizing that was in red and thus no longer there. This is fine!